integration-tests/itest/cli/cli.go

```diff
@@ -0,0 +1,191 @@
+package cli
```

Comment:
Is this part of the test or part of our testing framework?
Could we make the distinction more strict?

Reply:
Are these utils for testing the cli? Tests for the cli?

Reply:
I think that becomes clear with the usage
```
cli.Deploy(swissKnifeApp, jsonToken)
...
cli.ListApps()
...
cli.Status(swissKnifeApp)
```

Reply:
[very minor, personal opinion only] I do indeed understand it from its call-site usage. It is clear there.
From a code browsing and file/folder structure perspective I would personally have changed it from `cli/cli.go` to `testkit/cli.go`, `testkit/cli/cli.go` or some such similar indication of purpose.